zerver/openapi/zulip.yaml

```diff
@@ -16755,6 +16780,17 @@ paths:
                     the `PATCH /settings/display` endpoint.
                   type: boolean
                   example: true
+                typing_notifications_enabled:
+                  description: |
+                    Present if the user has chosen to receive typing notifications from
+                    other users.
```

Comment:
This seems wrong -- isn't it present if they have the `realm` object?

Reply:
updated.